docs/features/software-templates/writing-templates.md

```diff
@@ -533,6 +533,11 @@ catalogFilter:
     metadata.annotations.github.com/team-slug: { exists: true }
 ```
 
+#### Custom validation messages
+
+You may specify custom JSON Schema validation messages as supported by the
+[ajv-errors](https://github.com/ajv-validator/ajv-errors) plugin library to [ajv](https://github.com/ajv-validator/ajv).
```

Comment:
Maybe using capital letters won't give this spelling issue?

Otherwise 'ajv' word has to be added to 'vale' exclusion list

Reply:
correct on both counts; thanks!